safe-money-xmlbf/src/Money/Xmlbf.hs

```diff
@@ -83,16 +90,16 @@ instance Xmlbf.ToXml Money.SomeDiscrete where
              , ("n", T.pack (show (numerator r)))
              , ("d", T.pack (show (denominator r)))
              , ("a", T.pack (show (Money.someDiscreteAmount sd))) ]
-    in [ Xmlbf.element' (T.pack "money-discrete") (fromList as) [] ]
+    in [ either error id $ Xmlbf.element' (T.pack "money-discrete") (fromList as) [] ]
```

Comment:
`Xmlbf.element` (without the `'`) now returns a list :)

Reply:
Doesn't that just hide if we're doing something wrong, as mentioned in the docs?

Reply:
You are right :sweat_smile: I meant to write this comment in a line of the test file, but it seems I wrote it here instead :) 

This is fine, don't worry about changing it. And maybe let's keep your new version of the code in the tests as well :)